qemu/tests/pxe_query_cpus.py

```diff
@@ -81,11 +81,15 @@ def run(test, params, env):
         bg.start()
         error_context.context("Query cpus in loop", logging.info)
         vm.resume()
+        status_check = vm.monitor.get_status()
         while True:
             count += 1
             try:
                 vm.monitor.info("cpus", debug=False)
-                vm.verify_status("running")
+                if status_check['status'] == 'running':
+                    pass
```

Comment:
Seems it can't run into following else clause hence. Maybe we can use the status_check only for s390x.

Reply:
Modified with s390 only